src/algorithms/policy_gradient/sac.jl

```diff
@@ -0,0 +1,204 @@
+export SACPolicy, SAC_policy_network, create_sac_policy_network
+
+using Random
+using Flux
+using Flux.Losses: mse
+using Distributions: Normal, logpdf
+
+#### SAC models
+struct SAC_policy_network
```

Comment:
`SAC_policy_Network`
->
`SACPolicyNetwork`

It would be better to use CamelCase to name structs.